src/firestore/firestore-internal.ts

```diff
@@ -63,6 +85,7 @@ export function getFirestoreOptions(app: App): Settings {
   const credential = app.options.credential;
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const { version: firebaseVersion } = require('../../package.json');
+  const preferRest = firestoreSettings?.preferRest;
```

Comment:
Is the `@google-cloud/firestore` okay with options being `undefined`?

Reply:
Yes, it will perfectly accept `undefined`. https://github.com/googleapis/nodejs-firestore/blob/b1a0313498b0c3569bd74ce96fce80b8f9bee1a9/dev/src/index.ts#L560-L562